repo_db_mirror.go

```diff
@@ -192,7 +192,30 @@ func downloadAndLoadDB(mirror MirrorDB) error {
 		}
 		repoList = append(repoList, rpkg)
 	}
-	prefetchDB.Save(&repoList)
+	if db := prefetchDB.Save(&repoList); db.Error != nil {
+		if strings.Contains(fmt.Sprint(db.Error), "too many SQL variables") {
```

Comment:
go idiomatic way is to check for error and handle it in `if{}` branch. So to make it just negate if to

```go
if !strings.Contains(fmt.Sprint(db.Error), "too many SQL variables") {
  return db.Error
}

// normal case goes here
```